src/lib/mina_metrics/mina_metrics.ml

```diff
@@ -8,6 +8,8 @@ open Async_kernel
 
 let time_offset_sec = 1609459200.
 
+[%%inject "block_window_duration", block_window_duration]
```

Comment:
I'm on a mission to reduce our use of these, can you use `Mina_compile_config` instead?

Reply:
There is a dependency cycle  `Mina_compile_config -> currency -> snark_params ->...->allocation_functor -> mina_metrics`